#constants for Standby State Actions
uint8 WAIT = 0
uint8 START_NEW_MISSION = 1
uint8 RESUME_PAUSED_MISSION = 2

#request
uint8 paused_action
---
bool result